src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+
 public class Duke {
+    static final String LINE = "\t____________________________________________________________";
```

Comment:
Good that there are constants for different commands so that it is easy to change everything later on.